services/github/github-r-package.service.js

```diff
@@ -49,7 +49,7 @@ module.exports = class GithubRPackageVersion extends (
     },
     {
       title: 'GitHub R package version (branch & subdirectory of monorepo)',
-      pattern: ':user/:repo:/:branch',
+      pattern: ':user/:repo/:branch',
```

Comment:
Typo that was actually surfacing as an uncaught bug (see below screenshot from prod) that now gets surface by the example checks we have

![image](https://user-images.githubusercontent.com/13042488/102680521-805a0500-417e-11eb-9fd8-b0d0e46f8b11.png)